azurelinuxagent/common/logger.py

```diff
@@ -135,13 +138,23 @@ def write_log(log_appender):  # pylint: disable=W0612
             msg = msg_format.format(*args)
         else:
             msg = msg_format
+
+        # using local method here to avoid import textutil cyclic dependency
```

Comment:
we shouldn't duplicate this.

to break the cycle, refactor to another module that textutil and logger can use